images/virtualization-artifact/pkg/controller/vm/internal/statistic.go

```diff
@@ -138,6 +139,8 @@ func (h *StatisticHandler) syncResources(changed *virtv2.VirtualMachine,
 
 		memoryOverhead := memoryPodRequest.DeepCopy()
 		memoryOverhead.Sub(memoryKVVMIRequest)
+		mi := int64(1024 * 1024)
+		memoryOverhead = *resource.NewQuantity(int64(math.Ceil(float64(memoryOverhead.Value())/float64(mi)))*mi, resource.BinarySI)
```

Comment:
```suggestion
		memoryOverhead.RoundUp(resource.Mega)
```

Reply:
RoundUp is not working
https://go.dev/play/p/M7hpbmd61Ee

Reply:
:okay: